sdk/tests/conformance/extensions/ext-sRGB.html

```diff
@@ -125,9 +125,7 @@
   var err = gl.getError();
   if (err == gl.NO_ERROR) {
     testFailed("should NOT be able to create type " + formatName);
-  } else if (err == gl.INVALID_OPERATION) {
-    testFailed("should return gl.INVALID_ENUM for type " + formatName);
-  } else if (err == gl.INVALID_ENUM) {
+  } else if (err == gl.INVALID_ENUM || err == gl.INVALID_OPERATION) {
```

Comment:
Looking more closely, it looks like INVALID_VALUE could be generated here since internalformat isn't an accepted format, but INVALID_OPERATION should not be able to be generated.